### DISCUSSION TITLE
How to contact mapsModelsImporter developers?

### DISCUSSION THREAD

**Discussion by Solstav** [community/community#34754] [Other Feature Feedback, Questions, & Ideas]
Got a message in Blender while importing *rcv* google maps 3d picture to blender. the message said to contact the developers of MapsModelsImporter with full console log. I got the console log but can't reach for the developers.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/eliemichel/MapsModelsImporter? In that case you should open an issue *in that repository* so the developers see it. I notice there's a "troubleshooting" section in the repository README, you might want to check that first. :slightly_smiling_face:

**Reply by queenofcorgis** [reply]
Like @airtower-luna said, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
